pkgs/tools/security/der-ascii/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule {
+  pname = "der-ascii";
+  version = "unstable-2022-01-18";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "der-ascii";
+    rev = "a31891b88eb633a70d69e413cb4061a2576ebecc";
+    sha256 = "0pm368nim12akp55x13nk8vzjw92yzj27jvqsckgbzws5vbgrvld";
+  };
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
```

Comment:
#177698
```suggestion

ldflags = [ "-s" "-w" ];

```

Reply:
I see, but should we really be doing this for every Go package individually? Won’t this be a hassle to roll back once the Go support code gets proper settings for it? I’m not sure how debug info in Nixpkgs works and if you can ask for it to be installed.

Reply:
When this issue will be resolved, there will be a tree-wide removal of these flags. For the time being adding these flags manually is highly suggested. Nixpkgs won't supply separate debug symbols for packages, it was discussed on occasion here and there.